testing/src/main/java/io/stargate/it/http/MetricsTest.java

```diff
@@ -66,7 +67,13 @@ public void restApiHttpRequestMetrics() throws IOException {
             .get()
             .addHeader(TagMeHttpMetricsTagProvider.TAG_ME_HEADER, "test-value")
             .build();
-    Response execute = client.newCall(request).execute();
+
+    MutableInt status = new MutableInt();
+    try (Response execute = client.newCall(request).execute()) {
+      assertThat(execute.body()).isNotNull();
+      assertThat(execute.code()).isNotZero();
+      status.setValue(execute.code());
```

Comment:
Does `status` have to be a `MutableInt`? It appears to be accessed only by the test thread, so I guess it cab be a simple `int`. WDYT?

Reply:
The problem is it's used in a lambda below so it needs to be final. I suggested using a helper to with return value so it's not mutated.

Reply:
Would it make sense to extract the response reading code into a method in this case and write to `status` on the declaration line?

Reply:
Also, please add a comment about the need to full read and close the response :) Same as what we have in the PR, but it would be nice to have it in code too, please :)

Reply:
`int` will work, I can extract...